src/main/java/controllers/PostPresenter.java

```diff
@@ -0,0 +1,9 @@
+package controllers;
+
+public class PostPresenter {
+
+    public void createPost() {
+    }
+
+    public void displayPost() {}
```

Comment:
Good idea including this method, maybe it can take in a Post as a param and return some data structure containing info from that Post (maybe it could return a Post? I'm not sure to be honest)